app/src/main/java/com/gaurav/avnc/ui/prefs/PrefsActivity.kt

```diff
@@ -117,6 +118,9 @@ class PrefsActivity : AppCompatActivity(), PreferenceFragmentCompat.OnPreference
                 enableIf { it["gesture_style"] != "touchpad" }
                 disabledStateSummary = getString(R.string.pref_gesture_action_move_pointer)
             }
+            findPreference<ListPreference>("gesture_swipe3")!!.apply {
+                enableIf { it["gesture_style"] == "touchpad" }
+            }
```

Comment:
I think you meant to use `gesture_tap3`  here.

Nevertheless, I don't think you should disable three-finger tap for Touchscreen mode. You can set the default action to `None` if you are worried about aim precision.